src/renderer/store/modules/utils.js

```diff
@@ -175,6 +175,91 @@ const actions = {
     }
   },
 
+  async downloadMedia({ rootState, dispatch }, [url, title, extension, folder]) {
+    const usingElectron = rootState.settings.usingElectron
+    const askFolder = folder === ''
+    let fileHandler
+    const successMsg = `download ${title} is done`
```

Comment:
Downloading ${title} has finished/completed

Not sure which one is better

Reply:
For me, "Downloading ${title} has completed" sounds good.